programs/degen-pools/src/transfers.rs

```diff
@@ -0,0 +1,49 @@
+use anchor_lang::prelude::*;
+use crate::CustomError;
+
+#[derive(Accounts)]
+pub struct ExecuteTransaction<'info> {
+    #[account(mut)]
+    pub sender: Signer<'info>,
+    /// CHECK: Only used for SOL transfer
+    #[account(mut)]
+    pub receiver: AccountInfo<'info>,
```

Comment:
We don't need to receive `receiver` from the user's program invocation since we know that all transfers should be sent to our admin wallet which we have a reference to here: https://github.com/Degen-Markets/degen-markets-solana/blob/master/programs/degen-pools/src/constants.rs#L2